harness/determined/exec/gc_checkpoints.py

```diff
@@ -126,10 +126,10 @@ def main(argv: List[str]) -> None:
         help="Glob list to match against checkpoint list (JSON-formatted file)",
     )
```

Comment:
can we add a test case for tensorboard files being deleted by experiment deletion 

I'm kinda shocked we didn't have one. 

If its too much work, I think creating a ticket to do this is fine too.

Reply:
I think given the urgency of this ticket, we should merge this PR & I can start working on a separate one with tests

Reply:
can you create a Jira ticket for this?

Reply:
Created.